ethereum-network-partition-test/main_test.go

```diff
@@ -194,21 +232,17 @@ func partitionNetwork(t *testing.T, enclaveCtx *enclaves.EnclaveContext) {
 		partitionedNetworkServices[secondPartition][renderServiceId(clNodeValidatorIdTemplate, nodeIdForSecondPartition)] = true
 	}
 
-	partitionedNetworkConnections := map[enclaves.PartitionID]map[enclaves.PartitionID]enclaves.PartitionConnection{
+	partitionedNetworkConnections := map[enclaves.PartitionID]map[enclaves.PartitionID]float64{
 		firstPartition: {
-			secondPartition: blockedPartitionConnection,
+			secondPartition: blockedPartitionPacketLossPercentage,
 		},
 	}
```

Comment:
I guess we probably don't need all this logic anymore right? We can probably allocate each service to either partition1 or partition 2 from the start (since they don't move partition along the course of the test), and then we either switch from ALLOWED to BLOCKED or the other way around when we need

This will be more in sync with how starlark subnetworks are now working. Otherwise we're wondering why we're having all those complex loops here and in `repartitionNetworkUsingStarlark`

Reply:
I guess for the nodes to be on `partition1` and `partition2`, we would have to change the `eth2` package. Are we fine doing that? In my opinion, test specific logic we can put here (like `update_service`) and leave the package as generic as possible.

But I overall agree with the theme of simplifying this code!